grafana_loki_openapi.yaml

```diff
@@ -46,6 +46,56 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/QueryResponseBody"
+  /loki/api/v1/query_range:
+    get:
+      tags:
+        - Query Range
+      parameters:
+        - name: query
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: limit
+          in: query
+          schema:
+            type: integer
+            minimum: 1
+            default: 100
+        - name: start
+          in: query
+          schema:
+            type: integer
+        - name: end
+          in: query
+          schema:
+            type: integer
+        - name: step
+          in: query
+          schema:
+            type: string
+        - name: interval
+          in: query
+          schema:
+            type: number
+        - name: direction
+          in: query
+          schema:
+            enum:
```

Comment:
You can move `enum` and `default` into the `components.schema` section as `Direction` and replace this with ` $ref: "#/components/schemas/Direction"` so it can be reused in the `/query` endpoint as well.

Reply:
Done